src/pages/Homepage/HeroSlideshow.vue

```diff
@@ -111,7 +111,7 @@ export default {
 	serverCacheKey: props => {
 		let cacheKey = 'DefaultHeroSlideshow';
 		if (props.promoEnabled) {
-			cacheKey = 'PromoSlideshow';
+			cacheKey = 'PromoSlideshowPossibility';
```

Comment:
I have my serious doubts that this is even doing anything. If you look at getCacheKey, its doing a substring(0,8) so any getCacheKey parameter shorter than 8 chars will be trimmed anyway.... so:
cacheKey = 'PromoSlideshow'; and cacheKey = 'PromoSlideshowPossibility';  will actually return the same cacheKey of 'PromoSli'

Reply:
Reference: https://ssr.vuejs.org/guide/caching.html#page-level-caching we should really take a second look at this cache key stuff...

Reply:
No that's incorrect. The substring(0,8) only applies to the commit hash, which is then appended to the full key string.

Reply:
@eddieferrer I believe the substring will only be happen the ${UI_COMMIT} portion of that string.

Reply:
Whoops, misread that. You guys are right